tanner/sessions/session_analyzer.py

```diff
@@ -144,11 +144,13 @@ def find_location(ip):
         reader = Reader(TannerConfig.get('DATA', 'geo_db'))
         try:
             location = reader.city(ip)
+            if location.postal.code is None:
+                zcode = 0
             info = dict(
                 country=location.country.name,
                 country_code=location.country.iso_code,
                 city=location.city.name,
-                zip_code=location.postal.code,
+                zip_code=zcode,
```

Comment:
and what if the `location.postal.code` is not None? :)

Reply:
If it's none then it would be an integer right?

Reply:
oh wait, those are numbers but still passed as string. I'll do the conversion.

Reply:
`UnboundLocalError: local variable 'zcode' referenced before assignment`